delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -367,3 +387,202 @@ def list_alerts(self):
+
+        msg = _('Set performance_statistic_switch failed.')
+        self._assert_rest_result(result, msg)
+        self._assert_data_in_result(result, msg)
+        return result['data']
+
+    def _get_performance_strategy(self):
+        url = "/performance_statistic_strategy"
+        result = self.call(url, method='GET', log_filter_flag=True)
+
+        msg = _('Get performance_statistic_strategy failed.')
+        self._assert_rest_result(result, msg)
+        self._assert_data_in_result(result, msg)
+        return result['data']
+
+    def _set_performance_strategy(self, hist_enable=1, hist_duration=60,
+                                  auto_stop=0, duration=5, max_duration=0):
+        url = "/performance_statistic_strategy"
+        data = {
+            "CMO_STATISTIC_ARCHIVE_SWITCH": hist_enable,
+            "CMO_STATISTIC_ARCHIVE_TIME": hist_duration,
+            "CMO_STATISTIC_AUTO_STOP": auto_stop,
+            "CMO_STATISTIC_INTERVAL": duration,
+            "CMO_STATISTIC_MAX_TIME": max_duration
+        }
+        result = self.call(url, data, method='PUT', log_filter_flag=True)
+
+        msg = _('Set performance_statistic_strategy failed.')
+        self._assert_rest_result(result, msg)
+        self._assert_data_in_result(result, msg)
+        return result['data']
+
+    def _get_metrics(self, resource_type, resource_id, metrics_ids):
+        url = "/performace_statistic/cur_statistic_data"
+        params = "CMO_STATISTIC_UUID={0}:{1}&CMO_STATISTIC_DATA_ID_LIST={2}&"\
+            .format(resource_type, resource_id, metrics_ids)
+        return self.paginated_call(url, None, "GET",
+                                   params=params, log_filter_flag=True)
+
+    def enable_metrics_collection(self):
+        return self._set_performance_switch('1')
+
+    def disable_metrics_collection(self):
+        return self._set_performance_switch('0')
+
+    def configure_metrics_collection(self):
+        self.disable_metrics_collection()
+        self._set_performance_strategy(hist_enable=1, hist_duration=300,
+                                       auto_stop=0, duration=60,
+                                       max_duration=0)
+        self.enable_metrics_collection()
+
+    def get_pool_metrics(self, storage_id, selection):
+        pools = self.get_all_pools()
+        pool_metrics = []
+
+        select_metrics, select_ids = _get_selection(selection)
+        for pool in pools:
+            metrics = self._get_metrics(pool['TYPE'], pool['ID'],
+                                        select_ids)
+            for metric in metrics:
+                data_list = metric['CMO_STATISTIC_DATA_LIST'].split(",")
+                for index, key in enumerate(select_metrics):
+                    data = int(data_list[index])
+                    if key in consts.US_TO_MS_LIST:
+                        data = data * 1000
+                    labels = {
+                        'storage_id': storage_id,
```

Comment:
unit and lable fileds are required 
![image](https://user-images.githubusercontent.com/45681499/121462071-a1810d00-c9cd-11eb-9883-5ebc622f6363.png)

Reply:
Can you handle this label from driver.. Exporter cant assume unit and type